src/components/molecules/DropdownFiltered/DropdownFiltered.tsx

```diff
@@ -123,6 +122,7 @@ const DropdownFiltered = (props: DropdownFilteredProps) => {
                   endIcon={
                     theme.input.searchInput.customIcon ? (
                       <CustomIcon
+                        style={{ color: theme.input.searchInput.customIconColor }}
```

Comment:
can either do this or just set it to the colour hard coded in the CustomIcon component